### DISCUSSION TITLE
SO integrated within external Security Operation Center based on Splunk / QRadar

### DISCUSSION THREAD

**Discussion by mnasec** [Security-Onion-Solutions/securityonion#7930] [Unsupported Versions] [answered]
We are using SO 2.3.120 (actual 2 Forward, 2 Receiver, 8 Search and 1 Manager - Nodes) and plan to build up our own Security Operation Center and need to integrate for 24/7 operations an external SOCaaS. The external partner uses a Spluk or QRadar based solutions. The external partner is a big telecommunication player in europe and provides a SOCaaS for several customers, so we need at least some of their products to use the SOCaaS.

From the last security incident with quakbot and the unsuccessful try to implement a Cobalt-Strike beacon, we were able to quickly find and isolate the infected device with the running Security Onion solution. And we had a full protocol of the traffic and could provide the external incident response team many informations about the timeline and observables. So in sum: Security Onion is a great solution and helps a lot to keep the network save and find the problems. I do not want to miss my SO. Thanks to all who provide such a great solution for us!!!

Question for the community / SO Team: 
- Has someone a similar integration running and how are the different solutions connected?
- Which problems do you see by integrating such solution?
- Will Security-Onion-Solution support such a scenario via Premium Support / Professional Services?

**Comment by dougburks** [comment, marked as answer]
> * Will Security-Onion-Solution support such a scenario via Premium Support / Professional Services?

The best way to ask this question is to reach out to the team at https://securityonionsolutions.com/support to discuss your situation.
